drivers/sensor/vl53l0x/vl53l0x.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2017 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __SENSOR_VL53L0X_H__
```

Comment:
As this header is only used by vl53l0x.c, can we just put the defines & struct into vl53l0x.c and remove this file.